packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -289,3 +288,175 @@ def thread_request():
     assert len(set(expected_uuids)) == numapps
     assert len(set(received_uuids)) == numapps
     assert set(expected_uuids) == set(received_uuids)
+
+
+@pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
+def test_dcos_cni_l4lb(dcos_api_session):
+    '''
+    This tests the `dcos - l4lb` CNI plugins:
+        https: // github.com / dcos / dcos - cni / tree / master / cmd / l4lb
```

Comment:
Why is this url so weird?

Reply:
@orsenthil could you elaborate?

Reply:
I meant the spaces between `/` in the URL. Just oddity.